src/helpers/misc.js

```diff
@@ -172,6 +172,10 @@ const isDarklisted = addr => {
   return errObject;
 };
 
+const stripAddr = addr => {
```

Comment:
why arent we using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
I did not know about it.. My bad